widowx_envs/widowx_envs/widowx_env_service.py

```diff
@@ -137,13 +143,20 @@ def __move(self, pose: np.ndarray, duration: float):
                 eep = pose
             else:
                 eep = self.get_tf_mat(pose)
-            self.bridge_env.controller().move_to_eep(eep, blocking=False)
+
+            while True:
+                try:
+                    self.bridge_env.controller().move_to_eep(eep, blocking=True, duration=duration)
+                    self.bridge_env._reset_previous_qpos()
+                    break
+                except Environment_Exception as e: # TODO: return false if it isnt working
+                    print(e)
```

Comment:
Might want to make this return False in the client side, so the client can decide whether to re-execute the `move` command. I will make the change in my PR.